README.md

```diff
@@ -24,47 +24,57 @@ group of edits or changes, please use a process like git flow. If you are not
 familiar with this ask a dev or an admin to help you.
 
 ### 1st: Make your edit on github
-To edit a page first go to the info.arxiv.org page you want to edit. Ex https://info.arxiv.org/help/gzip.html
+- To edit a page first go to the info.arxiv.org page you want to edit. Ex https://info.arxiv.org/help/gzip.html
 
-![gzip page](non-info/help-info-page.png)
+- Click on the pencil icon to edit the page. That will take you to the corresponding page in github.
 
-There click on the pencil icon to edit the page. That will take you to the corresponding page in github.
+![Screenshot of a page on info.arxiv.org and the location of the edit icon](/non-info/02-help-info-page.png)
 
-![gzip page](non-info/help-github1.png)
+- On the page you want to edit in github, click the pencil to edit the page contents right in github.
 
-There click "edit" and you can edit contents of the page in github. Once you
-have made changes you can click preview to see how it will look when completed
-There are slight differences between the markdown we use and what
-github uses.
+![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)
 
-![gzip page](non-info/help-github2.png)
+- After you have made changes, click preview to see how it will look when completed. 
+**Please note:** There are slight differences between the markdown we use and what github uses.
```

Comment:
no idea what that means

Reply:
Github uses a different type of markdown than we do with mkDocs. It uses "Github flavored" markdown to comply with Common Mark which is a markup language which has extended syntax elements making it a "superset" of markdown.